app/controllers/SupportLandingPageController.scala

```diff
@@ -21,9 +22,24 @@ class SupportLandingPageController(
   runtime: zio.Runtime[ZEnv],
   dynamoTests: DynamoChannelTests,
   dynamoArchivedTests: DynamoArchivedChannelTests,
-  dynamoTestsAudit: DynamoChannelTestsAudit
+  dynamoTestsAudit: DynamoChannelTestsAudit,
+  permissionsService: DynamoPermissionsCache
 )(implicit ec: ExecutionContext) extends ChannelTestsController[SupportLandingPageTest](
-  authAction,
+  new AuthAndPermissionActions(
+    authAction,
+    // all users have read access
+    readPermissionAction = None,
+    // users must have write access to make changes
+    writePermissionAction = Some(
+      new PermissionAction(
+        page = SupportLandingPageController.name,
+        requiredPermission = Permission.Write,
+        permissionsService,
+        components.parsers,
+        ec
```

Comment:
I know the naming of this wasn't part of this change but It would be clearer to have `ExecutionContext`...maybe one to refactor at a later date

Reply:
👍 updated